crates/rolldown_common/src/types/watch.rs

```diff
@@ -56,10 +56,19 @@ impl From<BundleEventKind> for WatcherEventData {
   fn from(kind: BundleEventKind) -> Self {
     let mut map = HashMap::default();
     map.insert("code".to_string(), kind.to_string());
+    if let BundleEventKind::BundleEnd(data) = kind {
```

Comment:
The key could just use `&'static str`, you don't need to use `to_string`

Reply:
Updated.